package.json

```diff
@@ -34,5 +38,14 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
These shouldn't be under dev dependencies except for the test-specific ones since they are needed to build project.

Reply:
Hmmm, but it builds OK so I'm a bit confused. I thought devDeps was for stuff in the build pipeline (including test stff) that should not be included in the final build package. But I stand to be corrected on this!

Reply:
That's right, sorry for being confusing. I didn't know that Relay compiles and does all the stuff during the build! That's pretty cool. Ignore me!